### PR TITLE
Added blog post for 'feeling lost'

### PR DESCRIPTION
**Pull request by damiensedgwick**

### CODE REVIEW

File: src/pages/posts/2022-05-08-feeling-lost/index.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+title: "Feeling lost is not only for beginners"
+date: "2022-05-08T12:35:00Z"
+author: Damien Sedgwick
+tags: Sedgwick Short
+---
+
+Below are my current thoughts and feelings regarding development as a career.
+I wanted to share them publicly because I feel like it is important to show that 
+even though I have been a developer for a few years now, it is entirely normal
+to feel a little deflated and or burnt out.
```
Comment (tom-sherman) [inline src/pages/posts/2022-05-08-feeling-lost/index.mdx:12]:
```suggestion
to feel a little deflated or burnt out.
```

File: src/pages/posts/2022-05-08-feeling-lost/index.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Anyway, this is a kind of thoughts out loud post, but I feel like it would be 
+good to hear other peoples feelings and thoughts as I think only good can come 
+from showing some vulnerability from time to time and learning how to deal with
+those feelings and thoughts.
+
+I am at this moment in time, the most successful I have ever been in not just
```
Comment (tom-sherman) [inline src/pages/posts/2022-05-08-feeling-lost/index.mdx:19]:
```suggestion
I am, at this moment in time, the most successful I have ever been. Not just 
```

File: src/pages/posts/2022-05-08-feeling-lost/index.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+even though I have been a developer for a few years now, it is entirely normal
+to feel a little deflated and or burnt out.
+
+Anyway, this is a kind of thoughts out loud post, but I feel like it would be 
+good to hear other peoples feelings and thoughts as I think only good can come 
+from showing some vulnerability from time to time and learning how to deal with
+those feelings and thoughts.
+
+I am at this moment in time, the most successful I have ever been in not just 
+this career, but any career I have ever had and equally at the same time, I have
```
Comment (tom-sherman) [inline src/pages/posts/2022-05-08-feeling-lost/index.mdx:20]:
```suggestion
this career, but any career I have ever had but at the same time, I have
```

**Review by tom-sherman — COMMENTED** [review]
Some grammar nits. Also probs worth always contracting or always doing long form. Choose either "I've" or "I have" instead of mixing both too much.

File: src/pages/posts/2022-05-08-feeling-lost/index.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+never felt so lost in where to go or what to do next.
```
Comment (tom-sherman) [inline src/pages/posts/2022-05-08-feeling-lost/index.mdx:21]:
```suggestion
I am, at this moment in time, the most successful I have ever been. Not just in
this career but any career I have ever had. But at the same time I have
never felt so lost in where to go or what to do next.
```

File: src/pages/posts/2022-05-08-feeling-lost/index.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+general, I don't feel depressed as everything else seems to be going really well.
+
+A part of my issue is that I don't really want to work in web development forever, 
+or work on products that I don't really care about or believe in. I would 
+ultimately like to become an iOS engineer as I am an Apple fanboy deep down 
+(haters gonna hate), but I feel so stuck in the sense that changing from web
+developer to an iOS developer seems impossible, or at least it does for me at 
+the moment.
+
+Is it just me, or does anyone else feel lost?
```
Comment (tom-sherman) [inline src/pages/posts/2022-05-08-feeling-lost/index.mdx:45]:
```suggestion
Is it just me? Or does anyone else feel lost?
```

File: package-lock.json [hunk 6]
```diff
@@ -3318,7 +3318,6 @@
         "anymatch": "^2.0.0",
         "async-each": "^1.0.1",
         "braces": "^2.3.2",
-        "fsevents": "^1.2.7",
```
Comment (tom-sherman) [inline package-lock.json:3321]: Remove this diff?

Comment (damiensedgwick) [reply package-lock.json:3321]: I don't know why that is even in there 🤔

**Review by Illizian — APPROVED** [review]

**Pull request merged**